samples/system-test/containerAnalysis.test.js

```diff
@@ -0,0 +1,264 @@
+const { assert } = require('chai');
+const cp = require('child_process');
+const uuid = require(`uuid`);
+
+const grafeas = require('@google-cloud/grafeas');
+const client = new grafeas.v1.GrafeasClient();
+
+const execSync = cmd => cp.execSync(cmd, { encoding: 'utf-8' });
+const exec = cmd => cp.exec(cmd, {encoding: 'utf-8'})
+
+const projectId = process.env.GCLOUD_PROJECT;
+const formattedParent = `projects/${projectId}`;
+const uuidVal = uuid.v4();
```

Comment:
Thank you for writing this in a way that won't have collisions, and cleaning up resources after the first time.  I *really* appreciate that you took the time to do this the right way.